### PR TITLE
test(audio): cover AIFF reader edge cases

### PR DESCRIPTION
**Pull request by danielraffel**

### CODE REVIEW

File: test/test_audio_file.cpp [hunk 1]
```diff
@@ -2030,3 +2030,172 @@ TEST_CASE("AIFF reader rejects invalid COMM metadata and unsupported PCM depths"
...
+        comm[8] |= 0x80;
+
+        write_aiff_fixture(tmp_path,
+                           "AIFF",
+                           {
+                               {"COMM", comm},
+                               {"SSND", make_ssnd_chunk({0x40, 0x00})},
+                           });
+
+        REQUIRE_FALSE(registry.read_info(tmp_path.string()).has_value());
```
[the remark this answers is not in the document]
Comment (danielraffel) [reply test/test_audio_file.cpp:2174]: Fixed in a4fcb409af84de837da447c2423aef587b4289e6: AIFF sample-rate decoding now validates the extended value is finite, positive, and within uint32_t range before casting in both read_info() and read().
